solidity/contracts/converter/ConverterBase.sol

```diff
@@ -447,6 +454,25 @@ abstract contract ConverterBase is ConverterVersion, IConverter, TokenHolder, Co
         emit Conversion(_sourceToken, _targetToken, _trader, _amount, _returnAmount, int256(_feeAmount));
     }
 
+    /**
+     * @dev transfers funds held by the contract and sends them to an account
+     *
+     * @param token ERC20 token contract address
+     * @param to account to receive the new amount
+     * @param amount amount to withdraw
+     */
+    function safeTransfer(
+        IERC20 token,
+        address to,
+        uint256 amount
+    ) private {
+        if (token == NATIVE_TOKEN_ADDRESS) {
+            payable(to).transfer(amount);
+        } else {
+            token.safeTransfer(to, amount);
+        }
+    }
+
     /**
```

Comment:
I'd wrap everything with `if (amount > 0) { ... }`, as any one of the reserve tokens may require this condition inside its `transfer` function.
TBH, this risk also exists in many other places in our code, and specifically in this contract.
But this new function is one 'bottleneck' where we can get rid of that risk for several different flows.

Reply:
I'll add an early return, but note that we always try to avoid "wrapping everything" to improve code readibility

Reply:
early return is preferable over more indentations.